docs/source/guides/adaptive_batching.rst

```diff
@@ -3,4 +3,44 @@
 Adaptive Batching
```

Comment:
Should this be moved under Core Concepts?

Reply:
I'm fine with it here for now, but I don't have a strong opinion about this.